Battery-Notification/Battery-Notification.py

```diff
@@ -0,0 +1,17 @@
+import psutil
+from plyer import notification
+import time
+#From psutil we import sensors battery class which gives us battery percentage
```

Comment:
Instead of showing notification randomly after some time. You could do that only when the battery percentage changes. On way of doing it would be to measure the percentage before and after the while loop and check if they are unequal, if so show notification and make the vars equal. Feel free to implement it with your own logic too.

Reply:
@antrikshmisri the notification delay can be modified by changing the time delay.Also i think notification can be shown when the battery goes below a certain range like below 15 we can display warning to the user to charge or after 100 we can notify the user to unplug.
Is that what you are suggesting ?

Reply:
No, what I am trying to say is show the notification only when the battery percent decreases/increases. Currently, there is no logic for that the notification is shown after certain timeout. Add some logic for the same

Reply:
@antrikshmisri if after every increase or decrease if we show a notification then it would be distracting for the user i think.Else we could do like show the battery level and tell the user how many minutes or hours the battery can last on current level.What do you think?

Reply:
Hi @pankaj892,
Please update the PR with the changes suggested by @antrikshmisri so that we can send this PR to PA.

Reply:
@pankaj892 Sorry for the delay. You can add a certain timeout to delay the notifications.

Reply:
@antrikshmisri I have added a delay of 5 seconds so the notification pops up for every hour for 5 seconds.
Please review